genienlp/data_utils/bootleg.py

```diff
@@ -63,35 +63,36 @@ def __init__(self, args):
         self.almond_domains = args.almond_domains
         self.bootleg_post_process_types = args.bootleg_post_process_types
 
-        with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/qid2typenames.json', 'r') as fin:
+        with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/qid2typenames.json') as fin:
             self.qid2typenames = ujson.load(fin)
-        with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/type_vocab_to_wikidataqid.json', 'r') as fin:
-            self.type_vocab_to_wikidataqid = ujson.load(fin)
-            self.wikidataqid_to_type_vocab = {v: k for k, v in self.type_vocab_to_wikidataqid.items()}
-        with open(f'{self.args.database_dir}/es_material/typeqid2id.json', 'r') as fin:
+        with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/type_vocab_to_wikidataqid.json') as fin:
+            self.type_vocab_to_entityqid = ujson.load(fin)
+            self.entityqid_to_type_vocab = {v: k for k, v in self.type_vocab_to_entityqid.items()}
+        with open(f'{self.args.database_dir}/es_material/typeqid2id.json') as fin:
             self.typeqid2id = ujson.load(fin)
         self.id2typeqid = {v: k for k, v in self.typeqid2id.items()}
 
         ##### almond specific
-        with open(f'{self.args.database_dir}/es_material/almond_type_mapping_matching.json', 'r') as fin:
-            self.almond_type_mapping_match = ujson.load(fin)
-        with open(f'{self.args.database_dir}/es_material/almond_type_mapping_inclusion.json', 'r') as fin:
-            self.almond_type_mapping_include = ujson.load(fin)
-
-        # only keep subset for provided domains
-        self.type_mapping_match = dict()
-        self.type_mapping_include = dict()
-        for domain in self.almond_domains:
-            self.type_mapping_match.update(self.almond_type_mapping_match[domain])
-            self.type_mapping_include.update(self.almond_type_mapping_match[domain])
-
-        self.almond_type2qid = dict()
-        with open(f'{self.args.database_dir}/es_material/almond_type2qid.json', 'r') as fin:
-            almond_type2qid = ujson.load(fin)
-        for domain in self.almond_domains:
-            self.almond_type2qid.update(almond_type2qid[domain])
-        self.qid2almond_type = {v: k for k, v in self.almond_type2qid.items()}
-
+        self.almond_type_mapping = dict()
+        self.wiki2normalized_type = dict()
```

Comment:
Add a couple comments describing what are the keys and what are the values here.